src/Object/Assign.ts

```diff
@@ -36,9 +43,8 @@ Assign a list of [[Object]] into **`O`** with [[Merge]] (last-in overrides)
 ```ts
 ```
 */
-export type Assign<O extends object, Os extends List<object>, depth extends Depth = 'flat'> =
-    O extends unknown
-    ? Os extends unknown
-      ? _Assign<O, Os, depth>
-      : never
-    : never
+export type Assign<
+  O extends object,
+  Os extends List<object>,
+  depth extends Depth = 'flat'
+> = O extends unknown ? (Os extends unknown ? _Assign<O, Os, depth> : never) : never
```

Comment:
TypeScript conditional types should be kept multi-line, this notation is confusing on bigger conditionals.